theia/settings.py

```diff
@@ -82,43 +81,38 @@
 
 WSGI_APPLICATION = 'theia.wsgi.application'
 
-
 # Database
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'theia',
-        'USER': 'theia',
-        'PASSWORD': 'theia',
-        'HOST': 'postgres',
-        'PORT': 5432
+if os.environ.get('ENV') == "staging":
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': 'theia_staging',
+            'USER': os.getenv('STAGING_DB_USERNAME'),
+            'PASSWORD': os.getenv('STAGING_DB_PASSWORD'),
+            'HOST': 'zooniverse-staging.postgres.database.azure.com',
```

Comment:
I think the HOST should be an env var as well, best not to widely publicize our host dns (and they should be unavailable on the public network)

Reply:
Good call. I'll adjust that today. 

I was toying with the idea of making the whole object an env var, but I didn't know if it's easy to enter that on the deployment side...and doing the whole thing as six individual strings also seemed like it would get annoying for Ops :)

Reply:
> I was toying with the idea of making the whole object an env var, but I didn't know if it's easy to enter that on the deployment side...and doing the whole thing as six individual strings also seemed like it would get annoying for Ops :)

we will add public env data as strings via env variables and private data via secrets (like a file) in kubernetes. It's not a big deal with that many :)

Similar to these https://github.com/zooniverse/theia/blob/41024a5ac60f7de9ef15216268d732d5205baea4/kubernetes/deployment-staging.tmpl#L28-L34

Reply:
You could also create a URL that includes the host, user, pass, and db and store that as one single var. But it seems like that needs another lib (like django-environ or dj-database-url) to do it. Keeping a few env vars in the k8s deploy tmpl is totally fine.